app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_dictionary = {}
+
+    def inner(*args, **kwargs) -> any:
```

Comment:
any is the function, you can't use it for type hints. You have to use `typing.Any`. Read more: https://docs.python.org/3/library/typing.html#the-any-type